Dockerfile.template

```diff
@@ -9,6 +9,8 @@ COPY --from=artifacts /srv/repo/*.rpm /rpms/
 COPY --from=mcd /usr/bin/machine-config-daemon /binaries/machine-config-daemon
 
 FROM quay.io/coreos-assembler/fcos:testing-devel
+ARG FEDORA_COREOS_VERSION=412.36.0
```

Comment:
not sure what's the plan for this arg

Reply:
Ah, its useful to be built outside of Prow env. I think we should use `36.2022...` format, but yeah, that'd work too